interview-questions.md

```diff
@@ -8,36 +8,47 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: You can fix this by associating the person with the Cohort. Since the Cohort has the primary key, as indicated by has_many students, we have to associate the student that belongs_to Cohort. The foreign key should be on the Student side. The foreign key can be added my creating a variable to store cohort and assign the cohort a student. The student should have a foreign key that matches the Cohort's cohort_id, which is their primary key.
 
-Researched answer:
+Researched answer:Since the Cohort model has the has_many description, students should follow after has_many, and this means the Cohort model will hold the primary key. The Student model which belongs_to the Cohort model will hold the foreign keys, as the foreign key always takes the side that has "belongs_to". To create the foreign key, we set cohort to a variable for each student and input their name, which will assign the foreingn key in the database for Student. For example, golf = cohort.first golf.students.create student: "Tabi Trubey". 
```

Comment:
The cohort_id column would need to be added by generating a migration file to add it to the schema.